### DISCUSSION TITLE
求推荐基于VUE的低代码平台或无代码平台

### DISCUSSION THREAD

**Discussion by gavenwan** [taowen/awesome-lowcode#138] [General]
看见很多无代码/低代码平台，太多了眼花缭乱不会选择了，
求推荐满足以下条件的无代码平台
1）支持docker本地部署
2）支持企业内部使用（看到很多开源注明了企业使用收费）
*3）前端使用VUE构建，或者页面样式可以修改，这样容易定制界面（可选）
*4）主要用于公司内部信息管理系统构建，表格、表单、文档管理简易（可选）
*5）拖拽生成的表格，最好支持移动端样式（可选）

**Comment by LowCoderGitHub** [comment]
同学可以到低代码交流平台上了解下哦   https://www.lowcoder.com.cn/
